portal-web/test/functional/com/liferay/portalweb/macros/VideoShortcut.macro

```diff
@@ -58,6 +58,64 @@ definition {
 		PortletEntry.publish();
 	}
 
+	macro addCPInNonDefaultLanguage {
+		if (!(isSet(skipNavigation))) {
+			DMNavigator.gotoAddDocumentTypeCP(dmDocumentTypeName = "${dmDocumentTypeName}");
+		}
+
+		Type(
+			key_text = "${key_text}",
+			locator1 = "TextInput#ANY",
+			value1 = "${videoURL}");
+
+		Pause(locator1 = "5000");
```

Comment:
@liuxingye @WYZ-Issac  is this necessary? Is there a way we can use something other than a pause?

Reply:
@austinchiang I have removed the new macro

Reply:
woops missed it. Thanks!

Reply:
it is ok